draft-irtf-cfrg-aegis-aead.md

```diff
@@ -2896,15 +2898,15 @@ tag256 : b36a16ef07c36d75a91f437502f24f54
 # Acknowledgments
 {:numbered="false"}
 
-The AEGIS authenticated encryption algorithm was invented by Hongjun Wu and Bart Preneel.
+The AEGIS family of authenticated encryption algorithms was invented by Hongjun Wu and Bart Preneel.
 
 The state update function leverages the AES permutation invented by Joan Daemen and Vincent Rijmen. They also authored the Pelican MAC, which partly motivated the design of the AEGIS MAC.
 
 We would like to thank the following individuals for their contributions:
 
-- Eric Lagergren and Daniel Bleichenbacher for catching a broken test vector and Daniel Bleichenbacher for many helpful suggestions.
-- John Preuß Mattsson for his review of the draft, and for suggesting how AEGIS should be used in the context of DTLS and QUIC.
-- Bart Mennink and Charlotte Lefevre as well as Takanori Isobe and Mostafizar Rahman for investigating the commitment security of the schemes specified in this document.
-- Scott Fluhrer for his review of the draft as a member of the CFRG Crypto Panel.
+- Eric Lagergren and Daniel Bleichenbacher for catching a broken test vector, and Daniel Bleichenbacher for many helpful suggestions.
+- John Preuß Mattsson for his review of the draft and for suggesting how AEGIS should be used in the context of DTLS and QUIC.
+- Bart Mennink and Charlotte Lefevre, as well as Takanori Isobe and Mostafizar Rahman for investigating the committing security of the schemes specified in this document.
```

Comment:
Add a comma after `Rahman`, or we could list them like:

`Bart Mennink, Charlotte Lefevre, Takanori Isobe, and Mostafizar Rahman for`

Reply:
{ Bart, Charlotte } and { Takanori, Mostafizar } worked independently, so I wouldn't put them all in the same list.

A comma has been added after `Rahman`.